<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QWidget" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>650</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget#stackedWidget{
	background-color: rgba(255, 255, 255, 180);
	border-radius: 20px;
}</string>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4"/>
   <widget class="QWidget" name="page_5"/>
   <widget class="QWidget" name="page_6"/>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>115</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_2{
	background-color: rgba(255, 255, 255, 180);
	border-radius: 20px;
}

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,2,2,2,2,0,2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>当前管理员：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_useradmin">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stu_inform_guanlipushButton">
      <property name="text">
       <string>学生信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="class_managepushButton">
      <property name="text">
       <string>班级信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="course_manageButton">
      <property name="text">
       <string>课程信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectCourse_manageButton">
      <property name="text">
       <string>选课信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="scoremanagepushButton">
      <property name="text">
       <string>成绩信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Manager_Manage">
      <property name="text">
       <string>管理员账号管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>注销</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>widget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
